tools/metrics/maths.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+func ArithmeticMean(data []int64) (float64, bool) {
```

Comment:
Could be in `pkg` and could have a test, but outside this PR.

Maybe a quality of life cleanup PR later